libr/core/cmd_anal.c

```diff
@@ -8805,6 +8915,33 @@ static int cmd_anal_all(RCore *core, const char *input) {
 	case 'o': // "aao"
 		cmd_anal_objc (core, input + 1, false);
 		break;
+	case 'O': { // "aaO"
+		char *off = r_str_trim_dup (input + 2);
+		RAnalFunction *fcn;
+		RListIter *it;
+		if (off && *off) {
+			ut64 addr = r_num_math (NULL, off);
+			fcn = r_anal_get_fcn_at (core->anal, core->offset, 0);
```

Comment:
Define and assign in the same line

Reply:
It's used differently in both conditions, it might be better to leave it as it is instead of defining it twice.